app/models/workshop.rb

```diff
@@ -98,21 +94,45 @@ def location
     host.address.city rescue ''
   end
 
+  def time_zone
+    chapter.time_zone
+  end
+
   def self.policy_class
     WorkshopPolicy
   end
 
+  def date_and_time
+    return nil unless super
+    super.in_time_zone(time_zone)
+  end
```

Comment:
Rails assumes we want dates in GMT because that's the timezone set in the Rails config. Wrapping the `date_and_time` method means it's always in the correct time zone when we need it

Reply:
👍